src/main/scala/com/netflix/edda/mongo/MongoElector.scala

```diff
@@ -121,6 +121,7 @@ class MongoElector extends Elector {
           // if we got the update then we are leader and attempt to
           // archive the old leader record
           if (result == null) {
+            if (logger.isInfoEnabled) logger.info("Error becoming leader")
```

Comment:
I've updated this and will update the pull shortly.  It sounds like a good idea to change to parameterized calls for the entire app but I think that's out of scope for this change.